lib/translations.json

```diff
@@ -40,6 +40,9 @@
     "equality_assertion_failed_due_to_undetermination": "Equality cannot be determined. Both parts are undefined",
     "equality_assertion_failed_due_to_missing_property": "Equality check failed. Objects do not have %s property",
     "equality_assertion_failed_due_to_circular_references": "(circular references found, equality check cannot be done. Please compare objects' properties individually)",
+    "identity_assertion_be_equal_to": "Expected %s to be identical to %s",
+    "identity_assertion_be_not_identical_to": "Expected %s to be not equal to %s",
+    "identity_assertion_failed_due_to_undetermination": "Identity cannot be determined. Both parts are undefined",
```

Comment:
there's also spanish translation files that would be nice to keep updated :pray: